interface/STS-Provision.json

```diff
@@ -1932,12 +1853,9 @@
             "$ref": "#/definitions/RoleAssignment"
           }
         },
-        "ImpersonationAssignments": {
-          "description": "A list of impersonation assignments assigned to this Anvendersystem (service consumer).",
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/ImpersonationAssignment"
-          }
+        "CanImpersonate": {
+          "type": "boolean",
+          "description": "Specifies if STS will allow OnBehalfOf for all requests from this Anvendersystem (user system)"
```

Comment:
I would suggest 

"description": "Specifies if this Anvendersystem can impersonate other systems."

Reply:
updated